Clarify with commas (#31224)
vercel/next.js: docs/routing/introduction.md

export default Home
```

> In the example above, we have multiple links; each one maps a path (`href`) to a known page:

Reviewer: ```suggestion
In the example above, we have multiple links. Each one maps a path (`href`) to a known page:
```

> In the example above, we have multiple links. Each one maps a path (`href`) to a known page:

Reviewer 2: ```suggestion
The example above uses multiple links. Each one maps a path (`href`) to a known page:
```
